tests/resources/Util.robot

```diff
@@ -105,8 +105,9 @@ Cleanup VIC Appliance On Test Server
 
 Run VIC Machine Delete Command
     [Tags]  secret
-    ${output}=  Run  bin/vic-machine-linux delete --name=${vch-name} --target=%{TEST_URL} --user=%{TEST_USERNAME} --password=%{TEST_PASSWORD} --force=true --compute-resource=%{TEST_RESOURCE} --timeout %{TEST_TIMEOUT}
-    Run Keyword And Ignore Error  Should Contain  ${output}  Completed successfully
+    ${rc}  ${output}=  Run And Return Rc And Output  bin/vic-machine-linux delete --name=${vch-name} --target=%{TEST_URL} --user=%{TEST_USERNAME} --password=%{TEST_PASSWORD} --force=true --compute-resource=%{TEST_RESOURCE} --timeout %{TEST_TIMEOUT}
+    Should Be Equal As Integers  ${rc}  0
+    Should Contain  ${output}  Completed successfully
```

Comment:
I was also considering some sort of differential logic as well.  Like check before the delete happens and after and diff them and make sure it is as expected.